guides/common/modules/con_converting-a-host-to-rhel.adoc

```diff
@@ -0,0 +1,74 @@
+[id="converting-a-host-to-rhel_{context}"]
+= Converting a Host to {RHEL}
+
+You can convert {RHEL} derivative distributions into a supportable {RHEL} on a host while retaining installed applications and configurations.
+{Project} provides the *Convert2RHEL* utility to simplify the conversion process.
+
+The *Convert2RHEL* utility in {Project} consists of an Ansible role and Ansible playbook.
+You use the Ansible role to generate conversion data on {ProjectServer}, which includes enabling required repositories and creating products, activation keys, and host groups.
```

Comment:
@maximiliankolb I tried to word it differently than you were suggesting, pls check.

Reply:
Sounds good to me!